### DISCUSSION TITLE
Custom product boxes.

### DISCUSSION THREAD

**Discussion by LemonTop** [elementor/elementor#25873] [Developers Community]
HI EVeryone, i hope you are all well?  I'm trying to use wsdm labs custom product boxes (https://wisdmlabs.com/assorted-bundles-woocommerce-custom-product-boxes-plugin/) with elementor but it wasn't showing the boxes in place of the standard product info when we tried it on a product. I was just wondering if there was anything we can do to get it working? Their support said the following   Why CPB is not compatible with Elementor? Because elementor has elements. A product page is build using elements provided by elementor. The summary hook 'woocommerce_before_single_product_summary' which is required to display CPB product layout does not get executed when using elementor. Thus, CPB is not compatible with Elementor. To make CPB compatible the prouduct page should have one such hook on which We can display CPB content and remove the Elementors elements. 

 is there a way for me to add the before single product summary somewhere so i can get the plugin working?

**Comment by markd91** [comment]
I have the same question! Anyone?

**Comment by Martydesign** [comment]
Same here

**Comment by kjenti** [comment]
same here
